zfs-prune-snapshots

```diff
@@ -95,6 +98,11 @@ human() {
 	echo '0 seconds'
 }
 
+if ! type -P "zfs" &>/dev/null; then
```

Comment:
style nit: you can drop the quotes around `zfs`.  not a huge issue - this whole block of code is :+1: to me